attestation-agent/kbs_protocol/test/kbs-config.toml

```diff
@@ -1,6 +1,29 @@
+[http_server]
 insecure_http = true
-insecure_api = true
 sockets = ["0.0.0.0:8085"]
 
-[attestation_token_config]
-attestation_token_type = "CoCo"
\ No newline at end of file
+[attestation_token]
+insecure_key = true
+
+[policy_engine]
+policy_path = "/opa/confidential-containers/kbs/policy.rego"
+
+[attestation_service]
+type = "coco_as_builtin"
+work_dir = "/opt/confidential-containers/attestation-service"
+policy_engine = "opa"
+attestation_token_broker = "Simple"
+
+    [attestation_service.attestation_token_config]
+    duration_min = 5
+
+    [attestation_service.rvps_config]
+    store_type = "LocalFs"
+    remote_addr = ""
+
+[admin]
+insecure_api = true
+
+[[plugins]]
+name = "resource"
+type = "LocalFs"
```

Comment:
Need another line in the end `dir_path = "/opt/confidential-containers/kbs/repository"` to specify the resource dir.

Reply:
I put a PR to specify a default value if this field is not given to align with [the doc](https://github.com/confidential-containers/trustee/blob/main/kbs/docs/config.md#resource-configuration) where `dir_path` is not required. https://github.com/confidential-containers/trustee/pull/564

Reply:
thanks for the PR. I had this passing in my local tests and then decided to drop `dir_path` just before pushing since I noticed it was the same as the "default". That did not work well :-)